src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -39,6 +39,11 @@ public class AzureAppConfigurationOptions
         /// </summary>
```

Comment:
I see `Flag to indicate whether enable replica discovery` above.

That sentence looks worth fixing.